crates/router/src/core/payouts.rs

```diff
@@ -413,12 +413,12 @@ pub async fn payouts_update_core(
             helpers::make_payout_method_data(
                 &state,
                 req.payout_method_data.as_ref(),
-                payout_data.payout_attempt.payout_token.as_deref(),
-                &payout_data.payout_attempt.customer_id,
-                &payout_data.payout_attempt.merchant_id,
+                payout_data.payout_attempt.payout_token.clone().as_deref(),
+                &payout_data.payout_attempt.customer_id.clone(),
+                &payout_data.payout_attempt.merchant_id.clone(),
                 Some(&payouts.payout_type),
                 &key_store,
-                Some(&payout_data),
+                Some(&mut payout_data),
```

Comment:
@sahkal - we need to clone as we're passing a mutable reference to payout_data here